CONTRIBUTING.md

```diff
@@ -24,7 +19,8 @@ Making a Contribution
 1. Write a test which shows that the bug was fixed or that the feature works as expected.
 1. Send a pull request (PR) and work with us until it gets merged and published. Contributions may need some modifications, so a few rounds of review and fixing may be necessary. We will include your name in the ChangeLog :)
 1. For quick merging, the contribution should be short, and concentrated on a single feature or topic. The larger the contribution is, the longer it would take to review it and merge it.
-1. Mbed TLS is released under the Apache license, and as such, all the added files should include the Apache license header.
+1. Ensure that all new files include the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html) standard license header.
```

Comment:
We can't insist that _all_ files have the license header: some files can't have free-form text (mostly test data in binary form; we haven't put a license header in `.data` files in the test suite, but it's technically possible). Add something like “if possible”? Or keep “should”?

Reply:
OK